www/css/cytube.css

```diff
@@ -44,14 +43,6 @@
     margin-bottom: 0;
 }
 
```

Comment:
userlist and message buffer have been moved here and split into different classes